.github/CODEOWNERS

```diff
@@ -1858,6 +1858,7 @@
 /types/jsonstream/                              @Bartvds
 /types/jsontoxml/                               @benstevens48
 /types/jsonwebtoken/                            @SomaticIT @danielheim @brikou
+/types/jsonwebtoken-promisified                 @aneilbaboo
 /types/jspdf/                                   @amberjs
```

Comment:
Not a problem but FYI you don't need to modify this file yourself - it is supposed to be auto-generated